AnkiDroid/build.gradle

```
@@ -289,14 +289,14 @@ apply from: "../lint.gradle"
 dependencies {
     configurations.all {
         resolutionStrategy {
-            force 'org.jetbrains:annotations:24.0.1'
+            force 'org.jetbrains:annotations:24.1.0'
```

```suggestion
            // Timber has this as a dependency but they are not up to date. We want to force our version.
            force 'org.jetbrains:annotations:24.1.0'
```

I was curious what this was for as I'd forgotten - it's to avoid Timber's annotation dependency from transitively coming in and changing our desired version. They make no attempt to stay up to date so I think this still has value but the comment will be useful for future explorers